pallas-network/Cargo.toml

```diff
@@ -16,6 +16,8 @@ hex = "0.4.3"
 itertools = "0.10.5"
 pallas-codec = { version = "=0.20.0", path = "../pallas-codec" }
 pallas-crypto = { version = "=0.20.0", path = "../pallas-crypto" }
+pallas-addresses = { version = "=0.20.0", path = "../pallas-addresses" }
+pallas-primitives = { version = "=0.20.0", path = "../pallas-primitives" }
```

Comment:
we don't want to couple networking crate with addresses or primitives.

Reply:
done!